src/main/resources/application.properties

```diff
@@ -32,4 +32,6 @@ spring.mail.properties.mail.smtp.auth=true
 spring.mail.properties.mail.smtp.starttls.enable=true
 
 # Flyway Migration
-spring.flyway.locations=classpath:db/migration
\ No newline at end of file
+spring.flyway.locations=classpath:db/migration
+
+sprv.rbac.admin=ADMIN
```

Comment:
what is sprv?

Reply:
and where do this using, I can't find reference

Reply:
forgot to delete when testing, ahihi